# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1455828142 28800
#      Thu Feb 18 12:42:22 2016 -0800
# Node ID e577dfb37970e153b13068906083952db661ea9c
# Parent  31cd45258fc66723a35495d1294a41edcd128be4
* doc/doxyhtml/README: Update storage requirements, formatting.

diff --git a/doc/doxyhtml/README b/doc/doxyhtml/README
--- a/doc/doxyhtml/README
+++ b/doc/doxyhtml/README
@@ -1,11 +1,11 @@
 This directory contains documentation in Doxygen format for
 Octave's source code.  It is not created by default.
 
 To produce Doxygen documentation use
 
-make doxyhtml
+  make doxyhtml
 
 Doxygen documentation can be helpful for developers of Octave, but is not
 needed by users of Octave.  In addition, the documentation requires
-approximately 1.2GB of storage space.  For these reasons it is not maintained
+approximately 2 GB of storage space.  For these reasons it is not maintained
 under version control nor distributed in tarballs.
